src/styles/index.scss

```diff
@@ -1,3 +1,137 @@
 body {
   margin: 0;
+  font-family: Roboto, Arial, sans-serif;
+  font-style: normal;
+}
+
+.card {
+  margin: auto;
+  box-sizing: border-box;
+  background-color: rgba(255, 255, 255, 1);
+  width: 200px;
+  height: 406px;
```

Comment:
You don't need to use fixed height here. Just let the elements inside card change height.